data/remnant missions.txt

```diff
@@ -1252,3 +1252,240 @@ mission "Remnant: Expanded Horizons Quarg 3"
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 5"
+	name "Broken Jump Drive Delivery"
+	description "A team of engineers on <planet> would like a broken jump drive to study."
+	to offer
+		has "Remnant: Broken Jump Drive 4: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		attributes outfitter
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`You stop by the director's desk to report having salvaged another broken jump drive for one of their research teams. The director nods in appreciation and checks his list. "There is a new team starting a project on <planet>. Could you take it there please?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Much appreciated. The sooner they get it, the sooner they can start on their project. That being said, I have spoken with the other directors, and we have decided that we are getting so many requests that we will put these into the job board for you. Any time the landing scans detect one of these broken jump drives, the system will display a message on the job board listing where the next one needs to be delivered."`
+				accept
+			label refuse
+			`	He looks surprised, "Really? Oh. Let us know when you are ready to make the delivery. The researchers will be waiting for it."`
+				defer
+	on complete
+		conversation
+			`The landing pad on <planet> is tucked away just behind the spaceport. A handful of engineers have a conveyor belt already set up to offload the drive from your ship when you arrive. When you open the hatch they extend it right up the ramp, and quickly load the drive onto it. The machinery quickly whisks the drive into a nearby structure along with its escort. A couple of the engineers stop by to talk with you and hand you <payment>.`
+			`	"Thanks for bringing us this drive. It is a rare piece of technology for us to get our hands on."`
+			choice
+				`	"I will leave you to it."`
+					goto end
+				`	"What are you studying?"`
+			`	With a pleased expression they break into song about mining the deeps for the mysteries of the sky. As best you can tell, they are trying to figure out the materials that the drive is made with, followed by experiments to replicate those materials.`
+			label end
+			`	"Our experiments await. May your search be fruitful!" they trill as they head off.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		event "remnant research update bjd1" 100 200
+		"reputation: Remnant" ++
+
+
+
+event "remnant research update bjd1"
+# bjd1 = Broken Jump Drive 1. This event is included for forward compatibility with future missions that will require this event to be complete.
+
+
+mission "Remnant: Broken Jump Drive job"
```

Comment:
Doesn't this mission need a `to complete` tag that requires the player to have a broken jump drive?

Reply:
#2494 #1151

Reply:
Kind of round-about, but the on complete has `outfit "Jump Drive (Broken)" -1` and ES doesn't let missions put outfits into the negative, so it doesn't trigger until they have one to remove.